react/CircularChart/helpers.ts

```diff
@@ -0,0 +1,46 @@
+const makeThickness = (thickness: number, hasTwoBar: boolean): number => {
+  return hasTwoBar ? thickness : thickness * 1.33
+}
+
+export const makeSizeAndThickness = (
+  size: string,
+  hasTwoBar: boolean
+): object => {
+  switch (size) {
+    case 'small':
+      return {
+        primarySizeAndThickness: {
+          size: 85,
+          thickness: makeThickness(2, hasTwoBar)
+        },
+        secondarySizeAndThickness: {
+          size: 73,
+          thickness: 2.5
+        }
+      }
+
+    case 'medium':
+      return {
+        primarySizeAndThickness: {
+          size: 102,
+          thickness: makeThickness(1.9, hasTwoBar)
+        },
+        secondarySizeAndThickness: {
+          size: 88,
+          thickness: 2.2
+        }
+      }
+
+    default:
+      return {
+        primarySizeAndThickness: {
+          size: 150,
```

Comment:
Should the size be computed from the thickness in the case we use keyword to size the component? To better see the relation between all these numbers. And also because the expected size seems to be a bit different when there are two bars and not only one.

Reply:
j'étais parti là-dessus initialement. Mais je me suis dit que des valeurs en dures étaient plus judicieuse et facile à modifier/maintenir

J'ai aussi exploré la piste d'un transform scale avec une base commune mais ça posait souci pour les espacement notamment